src/views/index.html

```diff
@@ -19,14 +19,19 @@ <h2>ExpenseNG Twitter Bot</h2>
     <h3>Active API Endpoints</h3>
     <div class="content">
       <ol class="user-stories">
-        <li><code>GET [domain_url]/api/tweets/:query?</code><br>where <code>query</code> is a urlencoded string or comma seperated strings.</li>
-        <li><code>GET [domain_url]/oauth/authorize</code><br>a temporary endpoint for <code>@expenseng</code> to grant api access to thier Twitter account.</li>
+        <li><code>GET [domain_url]/api/tweets/:query?</code><br>where <code>query</code> is a urlencoded string or comma
+          seperated strings.</li>
+        <li><code>GET [domain_url]/oauth/authorize</code><br>a temporary endpoint for <code>@expenseng</code> to grant
+          api access to thier Twitter account.</li>
+        <li><code>GET [domain_url]/api/like</code><br>bot can Like tweets where it is tagged(hashtaged(#) or
+          mentions(@)) </li>
       </ol>
 
       <h3>Click to view</h3>
       <ul>
         <li><a href="api/tweets/%23expenseng">/api/tweets/%23expenseng</a></li>
         <li><a href="oauth/authorize">/oauth/authorize</a></li>
+        <li><a href="/api/like">bot can Like tweets where it is tagged(hashtaged(#) or mentions(@)) </a></li>
```

Comment:
remove this also, and implement `app.post('/api/timeline/mentions', like)` in `server.js`